app/views/ruleset/security_ruleset.html.erb

```diff
@@ -3,9 +3,74 @@
     <h1 class="govuk-heading-xl">Check your OpenAPI documents for security issues</h1>
   </div>
 
-  <% if @crunch_results.present? %>
-    <div class="govuk-grid-column-two-thirds">
-     <div><%= JSON.pretty_generate(@crunch_results) %></div>
+  <% if @score.present? %>
+    <div class="govuk-grid-column-full">
+
+        <div class="govuk-panel govuk-panel--confirmation">
+          <h1 class="govuk-panel__title">
+            API Security Check Complete
+          </h1>
+          <div class="govuk-panel__body">
+            Score: <strong><%= @score.round(2) %></strong>
+          </div>
+        </div>
+        <br/><br/>
+        <h2 class="govuk-heading-m">Summary</h2>
+        <dl class="govuk-summary-list">
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">
+              OpenAPI Validity:
+            </dt>
+            <dd class="govuk-summary-list__value">
+              <%= @openapi.capitalize %>
+            </dd>
+          </div>
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">
+              Issue Count:
+            </dt>
+            <dd class="govuk-summary-list__value">
+              <%= @counter %>
+            </dd>
+          </div>
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">
+              Criticality:
+            </dt>
+            <dd class="govuk-summary-list__value">
+              <%= @criticality %>
+            </dd>
+          </div>
+        </dl>
+
+        <br/><br/>
+
+        <table class="govuk-table">
+          <caption class="govuk-table__caption govuk-table__caption--m">Results</caption>
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <th scope="col" class="govuk-table__header">Criticality</th>
+              <th scope="col" class="govuk-table__header">Issue</th>
+              <th scope="col" class="govuk-table__header">Line Numbers</th>
+              <th scope="col" class="govuk-table__header">Remedy</th>
+            </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+          <% @issues.each do | issue |%>
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell"><%= issue[0] %></td>
+              <td class="govuk-table__cell"><%= issue[1] %></td>
+              <td class="govuk-table__cell"><%= issue[2].join(", ") %></td>
+              <td class="govuk-table__cell"><a href="https://www.gov.uk/guidance/gds-api-technical-and-data-standards">Details</a></td>
+            </tr>
+          <% end %>
+          </tbody>
+        </table>
+
+        <a href="<% root_path %>" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button">
+          Select Another Linter
+        </a>
```

Comment:
A helper method is available that would simplify this code:
```ruby
<%= govuk_link_to "Select Another Linter", root_path %>
```
Also, I'm sure this code as it is, is working as expected because I think:
```ruby
href="<% root_path %>"
```
Will output `href=""`

The code should be:
```ruby
href="<%= root_path %>"
```